src/modelio.cs

```diff
@@ -90,6 +90,28 @@ interface MDLAsset : NSCopying
 		[Export ("canExportFileExtension:")]
 		bool CanExportFileExtension (string extension);
 
+		[iOS (10,3), TV (10,2), Mac (10,12,4)]
+		[Export ("components", ArgumentSemantic.Copy)]
+		IMDLComponent[] Components { get; }
+
+		[iOS (10,3), TV (10,2), Mac (10,12,4)]
+		[Export ("setComponent:forProtocol:")]
+		void SetComponent (IMDLComponent component, Protocol protocol);
+
+		[iOS (10,3), TV (10,2), Mac (10,12,4)]
+		[Export ("componentConformingToProtocol:")]
+		[return: NullAllowed]
+		IMDLComponent IsComponentConforming (Protocol protocol);
```

Comment:
Yea so in the headers `setComponent:forProtocol:` and `componentConformingToProtocol:` have the exact same description:

> Extensible component support that allows user of ModelIO to customize MDLAssets to fit their format and workflow.

So I'm going for `GetComponent` instead of `IsComponentConforming`.